server/catgenome/profiles/dev/catgenome.properties

```diff
@@ -64,3 +64,6 @@ blat.search.url=http://genome.cse.ucsc.edu/cgi-bin/hgBlat
 blat.search.type=DNA
 blat.search.sort.order=query,score
 blat.search.output.type=psl
+
+#cache settings
+server.cache.enabled=true
```

Comment:
The property name looks a little ambiguous. Consider changing it, so it would be obvious, that this is an index cache. (e.g. server.index.cache.enabled).

Reply:
Changed in all files.